src/resolvers/Organization/image.ts

```diff
@@ -0,0 +1,13 @@
+import { BASE_URL } from "../../constants";
+import { Organization } from "../../models/Organization";
+import { OrganizationResolvers } from "../../types/generatedGraphQLTypes";
+
+export const image: OrganizationResolvers["image"] = async (parent) => {
+  const org = await Organization.findOne({
+    _id: parent._id,
+  });
+  if (org!.image) {
+    return `${BASE_URL}${org!.image}`;
```

Comment:
Hardcode this to `http://localhost:${process.env.PORT || 4000}/${org!.image}`.

Or if you want you can export `PORT` as a constant from `constants.ts` and use it here. Also, remember that this is just a temporary solution.